daemon.go

```diff
@@ -52,3 +52,15 @@ func stopDaemon(c *cli.Context) error {
 	// XXX TODO
 	panic("not implemented yet")
 }
+
+func stopCmd(c *cli.Context) error {
+	conf := contextToConfig(c)
+	fs := key.NewFileStore(conf.ConfigFolder())
+	var drand *core.Drand
+	drand, err := core.LoadDrand(fs, conf)
+	if err != nil {
+		slog.Fatal(err)
+	}
+	drand.Stop()
```

Comment:
Remember we're in client / daemon mode now, so you can't just load drand and then stop it, since it's already running. You have to issue a gRPC command to the daemon so that it stops itself. And if you do that, please do it in a separate PR so we have a better visibility :)